README.md

```diff
@@ -1,13 +1,25 @@
-# What is this?
+# Table of Contents
+- [Introduction](#introduction)
+- [Usage](#usage)
+- [Contributing](#contributing)
+  - [Contributions](#contributions)
+
+## Introduction
 
 This repository provides a collection of templates that help you get started with solving the Advent of Code puzzles in various programming languages.
 
 These examples are not meant as a 'best practice', but rather as a starting point for beginners: They are intended for people who are new to programming and don't know where to get started. The examples simply read the input from a file, and print the result to the console. You should be able to figure it out from there!
 
-# How to use
+## Usage
 
 Read the `README.md` file in the directory of the language you want to use.
 
-# Contributing
+## Contributing
 
 Want to add a template for a new language? Feel free to open a pull request!
+
+### Contributions:
+- C#, JavaScript, Python: _Kamiel de Visser_
+- Kotlin: _NoahMelle_
+- Dosato: _RobotNik08_
```

Comment:
My name is spelled wrong

it should be: "Robotnik08"

Reply:
Fixed it.